--- examples/creation~	Thu Dec  3 11:48:59 1998
+++ examples/creation	Sun Oct 17 13:56:54 1999
@@ -1,4 +1,4 @@
-#!/usr/bin/perl -I../
+#!/usr/bin/perl
 
 use Net::XWhois; 
 
--- examples/creation2~	Thu Dec  3 23:21:21 1998
+++ examples/creation2	Sun Oct 17 13:57:04 1999
@@ -1,4 +1,4 @@
-#!/usr/bin/perl -I../
+#!/usr/bin/perl
 
 use Net::XWhois; 
 use Data::Dumper; 
--- examples/whois~	Thu Apr  8 14:53:20 1999
+++ examples/whois	Sun Oct 17 13:57:14 1999
@@ -1,4 +1,4 @@
-#!/usr/bin/perl -I../
+#!/usr/bin/perl
 
 use Net::XWhois; 
 
